gxa/src/test/java/uk/ac/ebi/atlas/solr/analytics/baseline/BaselineAnalyticsFacetsReaderTest.java

```diff
@@ -14,7 +14,161 @@
 import static uk.ac.ebi.atlas.utils.GsonProvider.GSON;
 
 public class BaselineAnalyticsFacetsReaderTest {
-    private static final JsonObject RESPONSE_2_JSON_FACETS = GSON.fromJson(
+    private static final String JSON_RESPONSE =
```

Comment:
Any reason you decided to inline this? It looks a bit messy, it's probably easier to read as a proper JSON file.

Reply:
I think that in this case, because it’s only used once, inlining puts it closer to the actual use. If you look a the resources you need to look around a bit more to know who is responsible for keeping that file. For instance, if the test is removed or changed there’s some risk of cruft being left around.